### PR TITLE
Feature/core 746 add evan table

### PR DESCRIPTION
**Pull request by datrinh**
Add evan-table wrapper for bootstrap-table

### CODE REVIEW

File: dapps/evancore.vue.libs/src/components/table/table.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <b-table
+    v-bind="$attrs"
+    v-on="$listeners"
+    class="evan-table-wrapper"
+    :tbody-tr-class="'evan-table-body-row'"
+    :thead-tr-class="'evan-table-head-row'"
+    :thead-class="'evan-table-head'"
+  >
+    <!-- Pass on all named slots -->
+    <slot v-for="slot in Object.keys($slots)" :name="slot" :slot="slot" />
```
Comment (karladler) [inline dapps/evancore.vue.libs/src/components/table/table.vue:30]: linter is not complaining about missing key property here?

File: dapps/evancore.vue.libs/src/components/table/table.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+</script>
+
+<style lang="scss" scoped>
+@import '~@evan.network/ui/src/style/utils';
+
+.evan-table-wrapper {
+  margin: 0;
+  width: 100%;
+
+  /deep/ table.table.b-table {
```
Comment (karladler) [inline dapps/evancore.vue.libs/src/components/table/table.vue:51]: is the `/deep/` here necessary or could we maybe just add the `table-class` property to b-table and than apply the styles there?

**Review by karladler — APPROVED** [review]
pls check the comments before merging

File: dapps/evancore.vue.libs/src/components/table/table.vue (hunk 2 above)
Comment (datrinh) [reply dapps/evancore.vue.libs/src/components/table/table.vue:51]: It's necessary to modify created child components (`<table>` in this case)

File: dapps/evancore.vue.libs/src/components/table/table.vue (hunk 1 above)
Comment (datrinh) [reply dapps/evancore.vue.libs/src/components/table/table.vue:30]: Mine is not for some reason :D

Edit: Apparently slots are handles differently, so they don't require :key

**Pull request merged**